std_msgs/String[] object_labels
std_msgs/Int16[] object_classes
std_msgs/Float32[] object_confs
sensor_msgs/PointCloud2[] object_clouds
